impl/src/rendering/templates/crate.BUILD.template

```diff
@@ -5,6 +5,7 @@ cargo-raze crate build file.
 DO NOT EDIT! Replaced on runs of cargo-raze
 """
 
+# buildifier: disable=out-of-order-load
```

Comment:
for both `io_bazel_rules_rust` and `rules_rust`, it looks like this could be solved by moving the `bazel_skylib` load before `rules_rust`.

Is there a reason to go this way, rather than just rearranging them?

Reply:
Good call! Sorry about that, I should have spent more time on this. I've had many headaches solving buildifier defects in these templates so was just looking for the quickest solution 😅

Reply:
Just pushed some updates